tests/libs/Option.php

```diff
@@ -104,7 +105,7 @@ public static function create($value)
 	}
 
 
-	public function getIterator()
+	public function getIterator(): Traversable
```

Comment:
I ran into some deprecations while running tests on PHP 8.2, so took the liberty of fixing that.

Reply:
Interesting, my dev env runs on PHP 8.2 for quite some time already, tests on GitHub also include 8.2 🤔 But I guess it doesn't hurt to have it there 😊